plugins/@grouparoo/airtable/jest.config.js

```diff
@@ -0,0 +1,18 @@
+const { helper } = require("@grouparoo/spec-helper");
+
+module.exports = {
+  testEnvironment: "node",
+  testTimeout: helper.mediumTime,
+  maxWorkers: process.env.NOCK ? "1" : "50%",
```

Comment:
No. I needed it here because of the assumption that the table was empty in some tests, but is also used to write data into in others. This is a nice pattern if that turns out to be the case because I kept forgetting `--runInBand`